ansible_base/lib/abstract_models/common.py

```diff
@@ -167,17 +166,21 @@ def related_fields(self, request):
         basename = get_cls_view_basename(self.__class__)
 
         # Add any reverse relations required
+        missing_relations = []
         for relation in self._meta.related_objects + self._meta.many_to_many:
             field_name = relation.name
             # obey the model ignore list
-            # skip reverse m2m, we only want to manage associations via the forward
-            if field_name in self.ignore_relations or isinstance(relation, ManyToManyRel):
+            if field_name in self.ignore_relations:
                 continue
             reverse_view = f"{basename}-{field_name}-list"
             try:
                 response[field_name] = reverse(reverse_view, kwargs={'pk': self.pk})
             except NoReverseMatch:
-                logger.error(f"Wanted to add {reverse_view} for {self.__class__} but view was missing")
+                missing_relations.append(reverse_view)
+
+        if missing_relations:
+            if settings.DEBUG:
```

Comment:
For ultimate pedantry, I remember that SonarCloud likes for double-conditions to go in the same `if`

Reply:
100%, had to fix it for SQ!